src/main/java/nl/ou/debm/producer/CGenerator.java

```diff
@@ -509,9 +526,12 @@ public Function getFunction(int currentDepth, DataType type, Boolean withParamet
 
                     if(!newFunction.isCallable())
                         throw new RuntimeException(currentFeature.getClass().getName() + " does not produce a callable function after " + loopLimit + " tries");
-
+                    if(newFunction.getName().startsWith("function_")){
+                        System.out.println("ERROR");
+                    }
```

Comment:
This looks like a debug hook. Should it be removed? Or turned into an assert/throw RuntimeException?